mdtraj/core/topology.py

```diff
@@ -196,7 +199,9 @@ def __repr__(self):
         return "<%s at 0x%02x>" % (self._string_summary_basic(), id(self))
 
     def _string_summary_basic(self):
-        return "mdtraj.Topology with %d chains, %d residues, %d atoms, %d bonds" % (self.n_chains, self.n_residues, self.n_atoms, len(self._bonds))
+        return "mdtraj.Topology with %d chains, %d residues, \
+                %d atoms, %d bonds" % (self.n_chains, self.n_residues,
+                                       self.n_atoms, len(self._bonds))
```

Comment:
This should be changed.  All that gets cancelled is the `\n` -- the indentation spaces still get added to the returned string.  Change to this:

```Python
    def _string_summary_basic(self):
        return ("mdtraj.Topology with %d chains, %d residues, "
                "%d atoms, %d bonds" % (self.n_chains, self.n_residues,
                                        self.n_atoms, len(self._bonds)))